### PR TITLE
UserSettingsActivity Created

### PR DESCRIPTION
**Pull request by Dalo-Chinkhwangwa-Prof**
#Task 1
-Created UserSettingActivity

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".UserSettingActivity"></activity>
```
Comment (CraigVitirinyuT) [inline app/src/main/AndroidManifest.xml]: Please lock this orientation to portrait mode
Thank you

**Review by CraigVitirinyuT — CHANGES_REQUESTED** [review]
Please review my comments. Everything else looks great

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Dalo-Chinkhwangwa-Prof) [reply app/src/main/AndroidManifest.xml]: @CraigVitirinyuT thanks for the review - I made the changes please verify.

**Review by CraigVitirinyuT — APPROVED** [review]
Looks great. You can make the merge

**Pull request merged**